test/Interop/Cxx/class/Inputs/nested-records.h

```diff
@@ -53,4 +53,27 @@ struct HasForwardDeclaredNestedType {
 
 // TODO: Nested class templates (SR-13853).
 
+namespace Space {
+
+struct ForwardDeclaresFriend {
+  friend struct ForwardDeclaredFriend;
+  friend void takesFriend(struct ForwardDeclaredFriend f);
```

Comment:
```suggestion
  friend void takesFriend(ForwardDeclaredFriend f);
```
Not needed in C++.

Reply:
I know. This was part of the reproducer, IIRC it didn't crash without it. I can't verify this now that it no longer crashes, but I think we should keep it.